web_src/js/features/pull-view-file.js

```diff
@@ -68,11 +69,7 @@ export function initViewedCheckboxListenerFor() {
       const data = {files};
       const headCommitSHA = form.getAttribute('data-headcommit');
       if (headCommitSHA) data.headCommitSHA = headCommitSHA;
-      fetch(form.getAttribute('data-link'), {
-        method: 'POST',
-        headers: {'X-Csrf-Token': csrfToken},
-        body: JSON.stringify(data),
-      });
+      POST(form.getAttribute('data-link'), {data});
```

Comment:
I wonder what's the point of this POST. No `await` and all...

Reply:
It's simply to let the server know `the user has just viewed this file`.
There is no required request handling here, unless of course we want to `un-view` a file on an error code, which is currently not implemented yet.